cdap-ui/app/cdap/components/ResourceCenterButton/ResourceCenterButton.scss

```diff
@@ -27,10 +27,15 @@
     width: 58px;
     background: $success-message-bg;
     display: inline-flex;
+    display: -webkit-inline-box;
```

Comment:
This is the bug we talked about safari rendering it differently. The auto prefixer adds it above and so the default display that gets applied is inline-flex instead of -webkit-inline-box. Need to find a better to do this cross browser. Right now inline-flex and -webkit-inline-box is rendered different in safari alone.